sources/packages/backend/apps/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -210,10 +210,17 @@ export class ApplicationSummaryAPIOutDTO {
   studyStartPeriod: string;
   studyEndPeriod: string;
   id: number;
-  applicationName: string;
   status: ApplicationStatus;
   parentApplicationId: number;
+  /**
+   * Original application submission date.
+   */
   submittedDate?: Date;
+  /**
+   * Indicates if the application is able to use the
+   * change request feature. Other conditions may apply.
```

Comment:
Minor. I did not get the part `Other conditions may apply.`

Reply:
The property itself does not check other conditions, for instance, if the application is completed.
The idea is to indicate if the application has the possibility to execute the "new change request" if all conditions are met.